ensure_tiller_installed.go

```diff
@@ -196,7 +196,7 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 
 	{
 		o := func() error {
-			pod, err = getPod(c.k8sClient, tillerLabelSelector, c.tillerNamespace)
+			pod, err = getPod(c.k8sClient, c.tillerNamespace)
```

Comment:
Some tidying up to simplify here.

Arguably this should just be a method but I have limited time to invest and Tiller will be going away.